src/features/addUncheckAllToReplyToModal.ts

```diff
@@ -0,0 +1,66 @@
+import './addUncheckAllToReplyToModal.css';
+
+import {makeBTDModule} from '../types/btdCommonTypes';
+
+enum LABELS {
+  SELECT_ALL = 'Select All',
+  UNSELECT_ALL = 'Unselect All',
+}
+
+export const addUncheckAllToReplyToModal = makeBTDModule(({TD}) => {
+  const jsModalsContainer = document.querySelector('.js-modals-container') as HTMLDivElement;
+  /**
+   * Listen to modal that are appended to the modal container
+   *
+   * Idea is to check if it's the right modal, since it doesn't appear to have a uuid or other consistant identifier
+   */
+  const observer = new MutationObserver((mutationsList, observer) => {
+    if (mutationsList[0].addedNodes.length > 0) {
+      const addedNode = mutationsList[0].addedNodes[0];
+      const isReplyToModal =
+        document.querySelector('.js-reply-checkbox') instanceof HTMLInputElement;
+
+      if (isReplyToModal) {
+        const containerForNewCheckbox =
+          addedNode.childNodes[1].childNodes[5].childNodes[0].childNodes[1];
+
+        const label = document.createElement('label');
+        label.textContent = LABELS.UNSELECT_ALL;
+        label.setAttribute('for', 'uncheck_all_reply_to_modal_checkbox');
```

Comment:
If we're using an enum for the text content, we might as well use a variable for that id to prevent typos in the future.